config.json

```diff
@@ -34,7 +34,7 @@
       "description": "Specifies the distro rid for Unix OS.",
       "valueType": "property",
       "values": [],
-      "defaultValue": "${OSRid}"
+      "defaultValue": "${OSRid}-${CPUArch}"
```

Comment:
Who sets OSRid?  Where does that value come from?

---
In reply to: [115813333](https://github.com/dotnet/core-setup/pull/2342#discussion_r115813333) [](ancestors = 115813333)